CHANGELOG.md

```diff
@@ -534,3 +534,4 @@ Released with 1.0.0-beta.37 code base.
 ### Fixed
 -  Fix jsonrpc payload and response types (#4743) (#4761)
 -  Allowed more flexibility in typing the overly constrained `provider.disconnect` function (#4833)
+-  Fix bug in handleRevert logic for eth_sendRawTransaction (#4902)
```

Comment:
@drub0y could you add this under 1.7.3. Thanks

Reply:
Sure, done!